Packages/com.mmmaellon.lightsync/package.json

```diff
@@ -15,6 +15,9 @@
     "vpmDependencies": {
         "com.vrchat.worlds": "^3.5.2"
     },
+    "legacyPackages": [
+        "com.mmmaellon.smartobjectsync"
```

Comment:
This will tell the VCC to remove SmartObjectSync from the project, automatically, if it is detected in your Project in favor of LightSync.

Reply:
It's not a 1:1 replacement yet. SmartObjectSync has a lot of helper scripts I haven't recreated yet